apps/web/src/store/modules/user/index.ts

```diff
@@ -20,7 +22,7 @@ const state: UserState = {
     backend: storedUserState.backend,
     name: storedUserState.name,
     email: storedUserState.email,
-    language: storedUserState.language || 'en',
+    language: (storedUserState.language && languages[storedUserState.language]) ? storedUserState.language : 'en',
```

Comment:
description: This is the conversion code which makes work with both, 'ja' and 'jp'.